gallery.rb

```diff
@@ -0,0 +1,69 @@
+#Photo Gallery - Command Line Interface
+#
+#This program takes a set of images and generates an HTML image gallery
+#
+#Usage:
+# $ ruby gallery.rb photos/bunny-1.jpg photos/bunny-2.jpg photos/bunny-3.jpg
+#
+
+def html_template(image_tag)
```

Comment:
I'd give the argument a less specific name, like `content` or `body_html`.  There's nothing in this method itself that requires or assumes that an image tag is being passed in, so there's a good chance you'll use the method to handle things other than images in the future.  It winds up being confusing if you're passing in something other than an image tag but the argument is called `image_tag`.